src/main.cpp

```diff
@@ -901,6 +962,16 @@ bool CheckTransaction(const CTransaction& tx, CValidationState &state)
     return true;
 }
 
+static uint64_t CalcDefaultBlockPrioritySize() {
+    uint64_t nConsensusMaxSize = Params().GetConsensus().MaxBlockSize(GetAdjustedTime(), sizeForkTime.load());
+    // How much of the block should be dedicated to high-priority transactions,
+    // included regardless of the fees they pay. This is to help people who want
+    // to make free transactions and don't mind waiting a while: coin age stands
+    // in for the monetary value of the fee. Defaults to an arbitrary 5% of the
+    // current max block size.
```

Comment:
nit: my understanding was that it primarily functions as a mitigation against spam attacks and isn't explicitly to allow free transactions (I could be wrong, @gavinandresen would know better than me.) I only bring it up because people sometimes read too much into comments.